spyder_kernels/comms/tests/test_comms.py

```diff
@@ -0,0 +1,158 @@
+from spyder_kernels.comms.kernelcomm import KernelComm
+
+
+# =============================================================================
+# Constants
+# =============================================================================
+FILES_PATH = os.path.dirname(os.path.realpath(__file__))
+TIMEOUT = 15
+
+TKINTER_INSTALLED = False
+try:
+    import tkinter
+    TKINTER_INSTALLED = True
+except:
+    pass
+
+# =============================================================================
+# Fixtures
+# =============================================================================
+@pytest.fixture
+def kernel(request):
+    """Console kernel fixture"""
+    # Get kernel instance
+    kernel = get_kernel()
+    kernel.namespace_view_settings = {'check_all': False,
+                                      'exclude_private': True,
+                                      'exclude_uppercase': True,
+                                      'exclude_capitalized': False,
+                                      'exclude_unsupported': True,
+                                      'excluded_names': ['nan', 'inf',
+                                                         'infty',
+                                                         'little_endian',
+                                                         'colorbar_doc',
+                                                         'typecodes',
+                                                         '__builtins__',
+                                                         '__main__',
+                                                         '__doc__',
+                                                         'NaN', 'Inf',
+                                                         'Infinity',
+                                                         'sctypes',
+                                                         'rcParams',
+                                                         'rcParamsDefault',
+                                                         'sctypeNA', 'typeNA',
+                                                         'False_', 'True_'],
+                                      'minmax': False}
+    # Teardown
+
+    def reset_kernel():
+        kernel.do_execute('reset -f', True)
+    request.addfinalizer(reset_kernel)
+    return kernel
+
+
+@pytest.fixture
+def comms(kernel):
+    """Get the comms"""
+    commA = dummyComm()
+    commB = dummyComm()
+    commA.other = commB
+    commB.other = commA
+
+    frontend_comm = FrontendComm(kernel)
+    kernel_comm = KernelComm()
+
+    kernel_comm._register_comm(commA)
+
+    # Bypass the target system as this is not what is being tested
+    frontend_comm._comm_open(commB, {'content': None})
+
+    return (kernel_comm, frontend_comm)
+
+# =============================================================================
+# Tests
+# =============================================================================
+
+
```

Comment:
Please remove these blank lines.

Reply:
Should I not leave two white spaces between two functions?